crates/time/src/lib.rs

```diff
@@ -0,0 +1,319 @@
+use minicbor::{Encode, Decode};
+
+type Slot = u64;
+
+#[derive(Clone, PartialEq, Eq)]
+pub struct Bound {
+    pub time: u64, // Milliseconds
```

Comment:
suggestion: either rename to `time_ms` or have a `type Millis = u64` to remove the comment